etc/config/c++.amazon.properties

```diff
@@ -493,7 +493,7 @@ compiler.clang2010.ldPath=${exePath}/../lib|${exePath}/../lib/x86_64-unknown-lin
 compiler.clang2010.debugPatched=true
 
 
-group.clangx86trunk.compilers=clang_trunk:clang_assertions_trunk:clang_concepts:clang_p1144:clang_autonsdmi:clang_bb_p2996:clang_p2998:clang_p3068:clang_p3309:clang_p3367:clang_p3372:clang_p3385:clang_p3412:clang_implicit_constexpr:clang_lifetime:clang_p1061:clang_parmexpr:clang_patmat:clang_embed:clang_dang:clang_reflection:clang_variadic_friends:clang_widberg:clang_resugar:clang_clangir:clang_dascandy_contracts:clang_ericwf_contracts:clang_p1974:clang_chrisbazley
+group.clangx86trunk.compilers=clang_trunk:clang_assertions_trunk:clang_concepts:clang_p1144:clang_autonsdmi:clang_bb_p2996:clang_p2998:clang_p3068:clang_p3309:clang_p3367:clang_p3372:clang_p3385:clang_p3412:clang_p3776:clang_implicit_constexpr:clang_lifetime:clang_p1061:clang_parmexpr:clang_patmat:clang_embed:clang_dang:clang_reflection:clang_variadic_friends:clang_widberg:clang_resugar:clang_clangir:clang_dascandy_contracts:clang_ericwf_contracts:clang_p1974:clang_chrisbazley
```

Comment:
just adding the name doesn't configure this - you need to add a block below too. I'll do that though now, just a heads up for (maybe!?) in the future.